src/qiskit_iqm/fake_backends/chip_sample.py

```diff
@@ -0,0 +1,155 @@
+# Copyright 2022 Qiskit on IQM developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable=no-name-in-module,import-error,too-many-instance-attributes,too-many-arguments,line-too-long
+"""
+Abstract representation of an IQM chip sample.
+"""
+
+from typing import Union
+
+from .quantum_architectures import IQMQuantumArchitecture
+
+
+class IQMChipSample:
+    """
+    Class implementation to provide an abstract representation of the specifications of an IQM chip sample.
+    """
+
+    def __init__(
+        self,
+        quantum_architecture: IQMQuantumArchitecture,
+        t1s: dict[int, float],
+        t2s: dict[int, float],
+        one_qubit_gate_fidelities: dict[str, dict[int, float]],
+        two_qubit_gate_fidelities: dict[str, dict[tuple[int, int], float]],
+        one_qubit_gate_depolarization_rates: dict[str, dict[int, float]],
+        two_qubit_gate_depolarization_rates: dict[str, dict[tuple[int, int], float]],
+        one_qubit_gate_durations: dict[str, float],
+        two_qubit_gate_durations: dict[str, float],
+        id_: Union[str, None] = None,
+    ):
+        """Provides the specifications of a quantum chip sample.
+
+        Args:
+            quantum_architecture (IQMQuantumArchitecture): an instance of a Quantum Architecture.
+            t1s (dict[int, float]): T1 times for the qubit with the corresponding key.
+            t2s (dict[int, float]): T2 times for the qubit with the corresponding key.
+            one_qubit_gate_fidelities (dict[str, dict[int, float]]): for each one-qubit gate, a fidelity is specified for each qubit.
+            two_qubit_gate_fidelities (dict[str, dict[tuple[int, int], float]]): for each two-qubit gate, a fidelity is specified for each coupling.
+            one_qubit_gate_depolarization_rates (dict[str, dict[int, float]]): for each one-qubit gate, a depolarization rate is specified for each qubit.
+            two_qubit_gate_depolarization_rates (dict[str, dict[tuple[int, int], float]]): for each two-qubit gate, a depolarization rate is specified for each coupling.
+            one_qubit_gate_durations (dict[str, float]): for each one-qubit gate, a duration is specified.
+            two_qubit_gate_durations (dict[str, float]): for each two-qubit gate, a duration is specified.
+            id_ (Union[str, None], optional): the identifier of the chip sample. Defaults to None.
```

Comment:
1. We usually do not repeat the type of the argument in the docstring - it is enough to have it in the function signature. 
2. In general would be nice to polish the wording and explain in a bit more detail what are these arguments. 
3. All arguments should mention their unit.

Reply:
I un-resolved this since points 1 and 4 have not been addressed yet

Reply:
Adressed in [7849774](https://github.com/iqm-finland/qiskit-on-iqm/pull/35/commits/78497740e67759dbb44168045010df1d809c9750).